WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -399,6 +431,15 @@ private extension QuickStartTourGuide {
     private struct Constants {
         static let maxSkippedTours = 3
         static let suggestionTimeout = 10.0
+        static let localNotificationInterval: TimeInterval = 60*60*24*2
```

Comment:
Rather than doing date math ourselves here, we could use `dateByAddingComponents`. So, a constant for `localNotificationIntervalInDays = 2` (or something) and then:

```
let components = DateComponents(day: Constants.localNotificationIntervalInDays)
let triggerDate = Calendar.current.date(byAdding: components, to: Date())
```